### ISSUE TITLE
Prerender's proxy_pass rewrites to http only

### ISSUE THREAD

**Issue by JoshSmith** [tonycoco/heroku-buildpack-ember-cli#45]
The nginx.conf is set up so that it rewrites only to http URLs, which will yield a 301 for those of us using SSL.

**Comment by tonycoco** [comment]
Looking into this... have you turned on SSL with the `FORCE_HTTPS` Heroku var?

**Comment by jordpo** [comment]
we were having the same issue using prerender and a https redirect. it would be redirected back to the http page. a patch that we are using is replacing $scheme with https when FORCE_HTTPS is present and true on line 91 of the config file. 

```
          <% if ENV["FORCE_HTTPS"] && ENV["FORCE_HTTPS"] == "true" %>
            rewrite .* /https://$host$request_uri? break;
          <% else %>
            rewrite .* /$scheme://$host$request_uri? break;
          <% end %>
```

**Comment by JoshSmith** [comment]
This looks like a good solution @jordpo. @tonycoco how do you feel about implementing this directly?

**Comment by tonycoco** [comment]
Should work. PR?

**Comment by JoshSmith** [comment]
@jordpo would you mind creating a PR for this?

**Comment by jordpo** [comment]
Will do. Here it is https://github.com/tonycoco/heroku-buildpack-ember-cli/pull/59

**Comment by tonycoco** [comment]
Closed with #59
